tests/translator/input/api_with_auth_no_openapi_flag.yaml

```diff
@@ -0,0 +1,58 @@
+Globals:
```

Comment:
@keetonian @brettstack I need your recommendation here - If I dont have a global api flag and define the inline swagger as openapi, we still add the auth parts to this inline swagger in swagger 2.0 format. This is obviously the wrong thing to do.
1. We can either fail if during auth we check that this is "openapi" but does not have OpenApiVersion:3
2. We see that this has "openapi: 3" and not "swagger: 2" so we add auth in openapi format (securitySchemes)

What is your recommendation?